lms/djangoapps/ccx/utils.py

```diff
@@ -266,12 +268,31 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 errors.append(u"{0}".format(exp))
                 continue
 
-            if CourseEnrollment.objects.is_course_full(ccx_course_overview) and not must_enroll:
-                error = _(u'The course is full: the limit is {max_student_enrollments_allowed}').format(
-                    max_student_enrollments_allowed=ccx_course_overview.max_student_enrollments_allowed)
-                log.info(u"%s", error)
-                errors.append(error)
-                break
+            # Check the License limit in the pearson course operation plugin.
+            if (
+                    configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
+                    and is_ccx_course(course_key)
+                    and not must_enroll
+               ):
+    
+                allow_enrollment, message = run_extension_point(
+                    'PCO_ENFORCE_LICENSE_LIMITS',
+                    course_key=course_key,
+                )
+
+                if not allow_enrollment:
+                    log.info(u"%s", message)
+                    errors.append(message)
+                    break
+
+            else:
```

Comment:
This 'else' it's not necessary since the previous block is breaking the flow. @elvinquero 
Also, if allow_enrollment is True, we will skip the next section.